RLBotPack/Necto/bot.cfg

```
@@ -0,0 +1,30 @@
+[Locations]
+# Path to loadout config. Can use relative path from here.
+looks_config = ./appearance.cfg
+
+# Path to python file. Can use relative path from here.
+python_file = ./bot.py
+requirements_file = ./requirements.txt
+
+# Name of the bot in-game
+name = Necto
+
+# The maximum number of ticks per second that your bot wishes to receive.
+maximum_tick_rate_preference = 120
+
+[Details]
+# These values are optional but useful metadata for helper programs
+# Name of the bot's creator/developer
+developer = Rolv, Soren, and several compute contributors
+
+# Short description of the bot
+description = Necto is the official RLGym community bot, trained using PPO with workers run by people all around the world.
+
```

you can add tags like this so it can appear in the categories
```suggestion
tags = 1v1, teamplay

```